lightly/api/api_workflow_datasources.py

```diff
@@ -136,6 +145,8 @@ def download_raw_samples(
             relevant_filenames_file_name=relevant_filenames_file_name,
             use_redirected_read_url=use_redirected_read_url,
             progress_bar=progress_bar,
+            relevant_filenames_run_id=run_id,
```

Comment:
Should we check before that either both are `None` or none are `None`?

Reply:
Yup updated.